d8/d8forms/d8forms.module

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/**
+ * @file
+ * Contains d8forms.module..
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+
+define('D8FORMS_MODULE_NAME', 'd8forms');
```

Comment:
Because I need the module's name in various places (for example in watchdog()) I thought this could be a good idea the prevent typos in my code. I've never used this "pattern" before either.